search/search_indexes.py

```diff
@@ -30,6 +31,12 @@ def index_queryset(self, using=None):
         """Records to check when updating entire index."""
         return self.get_model().objects.all()
 
+    def prepare_text(self, obj):  # pylint: disable=no-self-use
```

Comment:
@Ferdi It's slightly different. If a `LearningResource` has `content_xml` and `title`, and you want to type in "protein" and match any `LearningResource` that has "protein" in either `content_xml` or `title`, then you have to index them together.

This is because:

> Every SearchIndex requires there be one (and only one) field with document=True. This indicates to both     Haystack and the search engine about which field is the primary field for searching within.

From [this](http://django-haystack.readthedocs.org/en/latest/tutorial.html#configuration) documentation.

Everything else we index apart from the one marked `document=True` is going to have to be a separate filter. In the case of a `LearningResource`, the `content_xml` is the "meat" of the object, so it would be the "document."